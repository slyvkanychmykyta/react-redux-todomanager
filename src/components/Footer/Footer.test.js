import React from 'react';
import * as renderer from 'react-test-renderer';
import {Footer} from './index';


describe('snapshot test', () => {
    test('should render component with no todos', () => {
        const props = {
            todos: [],
            setAllCompleted: jest.fn(),
        };

        const TodoFormComponent = renderer
            .create(<Footer {...props}/>)
            .toJSON();
        expect(TodoFormComponent).toMatchSnapshot();
    });
});
