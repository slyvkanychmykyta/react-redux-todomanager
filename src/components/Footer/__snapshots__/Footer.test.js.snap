// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test should render component with no todos 1`] = `
<footer>
  <div
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "display": "block",
        "height": 25,
        "width": 25,
      }
    }
  >
    <svg
      style={
        Object {
          "display": "block",
          "height": 25,
          "width": 25,
        }
      }
      version="1.1"
      viewBox="0 0 27 27"
    >
      <g>
        <rect
          fill="rgb(0, 0, 0)"
          height="25"
          rx="5"
          ry="5"
          stroke="rgb(0, 0, 0)"
          style={
            Object {
              "strokeWidth": 2,
            }
          }
          width="25"
          x="1"
          y="1"
        />
        <path
          d="M22.12 6c-3.12 3.16-6.84 6.36-10.23 9.64l-5.42-4.05L4 14.84l6.78 5.08L12.23 21l1.25-1.25C17 16.2 21.29 12.6 25 8.89z"
          fill="rgb(255, 255, 255)"
        />
      </g>
    </svg>
  </div>
  <span
    className="completeAllTasks"
  >
    Все задачи выполнены
  </span>
</footer>
`;
